templates/news/list.html.twig

```diff
@@ -4,249 +4,185 @@
-
-                                                    <!--[if IE 9]>
-                                                    <video style="display: none;>
-                                                    <![endif]-->
-                                                    {% if page.featuredImage.byName('news-size-s') %}
-                                                        <source media="(max-width: 320px)" srcset="{{ page.featuredImage.byName('news-size-s') }}" data-srcset="{{ page.featuredImage.byName('news-size-s') }}">
-                                                    {% endif %}
-
-                                                    {% if page.featuredImage.byName('news-size-m') %}
-                                                        <source media="(max-width: 768px)" srcset="{{ page.featuredImage.byName('news-size-m') }}" data-srcset="{{ page.featuredImage.byName('news-size-m') }}">
-                                                    {% endif %}
-
-                                                    {% if page.featuredImage.byName('news-size-l') %}
-                                                        <source media="(max-width: 1280px)" srcset="{{ page.featuredImage.byName('news-size-l') }}" data-srcset="{{ page.featuredImage.byName('news-size-l') }}">
-                                                    {% endif %}
-                                                    <!--[if IE 9]></video>
-                                                    <![endif]-->
-                                                    <!-- Fallback -->
-                                                    {% if page.featuredImage.byName('news-size-m') %}
-                                                        <img src="{{ page.featuredImage.byName('news-size-m') }}" alt="" role="presentation" aria-hidden="true">
-                                                    {% else %}
-                                                        <img src="{{ page.featuredImage.byName('full') }}" alt="" role="presentation" aria-hidden="true">
-                                                    {% endif %}
+            {% if pages.pagination.totalResults > 0  %}
+                {% if not filterSet and showFeature %}
+                    <div class ="news_div" >
+                        {% include 'news/block.html.twig' with {'page': pages.current} %}
+                    </div>
+                {% endif %}
+            {% endif %}
 
-                                                </a>
-                                            </picture>
+            
+            <div class="govuk-grid-row" >
+                <div class="govuk-grid-column-one-third">
+                    {% if categoriesFilters is not empty %}
+                        <div class="govuk-accordion ccs-accordion ccs-accordion--clean" data-module="accordion" id="audience-tag-accordion">
```

Comment:
Would change this and others from:

`data-module="accordion"`

to 

`data-module="govuk-accordion"`

This to ensure the javascript works in the new govuk update.